tipg/dbmodel.py

```diff
@@ -163,14 +163,30 @@ class Collection(BaseModel):
     default_tms: str = tile_settings.default_tms
 
     @root_validator
-    def bounds_default(cls, values):
+    def extent_default(cls, values):
         """Get default bounds from the first geometry columns."""
+        extent = {}
+
         geoms = values.get("geometry_columns")
         if geoms:
             # Get the Extent of all the bounds
             minx, miny, maxx, maxy = zip(*[geom.bounds for geom in geoms])
-            values["bounds"] = [min(minx), min(miny), max(maxx), max(maxy)]
-            values["crs"] = f"http://www.opengis.net/def/crs/EPSG/0/{geoms[0].srid}"
+            extent["spatial"] = {
+                "bbox": [[min(minx), min(miny), max(maxx), max(maxy)]],
+                "crs": f"http://www.opengis.net/def/crs/EPSG/0/{geoms[0].srid}",
+            }
+
+        dates = values.get("datetime_columns")
+        if dates:
+            intervals = []
+            for d in dates:
+                if d.min or d.max:
```

Comment:
@bitner right now we have no way to set min/max extent for datetime columns, I'm not sure there is an easy solution to this 🤷‍♂️

Reply:
@vincentsarago I've made PR #13  to this branch that adds the min/max to the datetime_columns.